app/experimenter/nimbus-ui/src/components/AppLayoutWithSidebar/mocks.tsx

```diff
@@ -21,7 +21,7 @@ export const Subject = ({
 
   return (
     <RouterSlugProvider mocks={[mock]} path="/my-special-slug/edit">
-      <AppLayoutWithSidebar {...{ experiment }}>
+      <AppLayoutWithSidebar {...{ experiment }} refetch={async () => {}}>
```

Comment:
Any reason `AppLayoutWithSidebar` couldn't also have this default refetch value?